fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -1153,6 +1158,10 @@ private void startMasterOnlyDaemonThreads() {
         // start routine load scheduler
         routineLoadScheduler.start();
         routineLoadTaskScheduler.start();
+        // start dynamic partition task
+        dynamicPartitionScheduler.setName("DynamicPartitionScheduler");
+        dynamicPartitionScheduler.setInterval(Config.dynamic_partition_check_interval_seconds * 1000L);
+        dynamicPartitionScheduler.start();
```

Comment:
Would better add a config to disable or enable this feature.

Reply:
> Would better add a config to disable or enable this feature.

Ok, I will add a dynamic_partition_enable config